app/controllers/v0/my_va/submission_statuses_controller.rb

```diff
@@ -24,6 +24,19 @@ def allowed_forms
         %w[20-10206 20-10207 21-0845 21-0972 21-10210 21-4142 21-4142a 21P-0847]
       end
 
+      def apply_pdf_support(result)
```

Comment:
Can this method be moved to the report class and called form within it?

Reply:
i can try it out 👍 . I was unsure if there would be problems if we were to import functionality from `/app` into a module in '/lib'.  Would there be any concerns with that?  If so, I might also be able to move the `SubmissionPdfUrlService` into `/lib/forms/submission_status` as well.

Reply:
I was thinking the `Report` class would depend on the `SubmissionPdfUrlService` class.  Let's hop on a call and hash it out.